presto-main/src/main/java/com/facebook/presto/util/MergeSortedPages.java

```diff
@@ -104,13 +105,13 @@ private MergeSortedPages() {}
         PageBuilder pageBuilder = new PageBuilder(outputTypes);
         return pageWithPositions.transform(pageWithPositionOptional -> {
             if (yieldSignal.isSet()) {
-                return ProcessorState.yield();
+                return TransformationState.yield();
```

Comment:
We don't need `TransformationState` prefix, do we?

Reply:
this is a lambda and lambda does not "see" default methods of the interface it's intended to implement